### PR TITLE
Add back code for sending email notification to newly-approved users

### PR DESCRIPTION
**Pull request by jacoblurye**
I think I accidentally deleted this functionality during #245's major refactor. A cautionary tale about having good test coverage before refactoring code!

### CODE REVIEW

File: cidc_api/models/models.py [hunk 1]
```diff
@@ -252,6 +253,14 @@ class Users(CommonColumns):
     role = Column(Enum(*ROLES, name="role"))
     disabled = Column(Boolean, default=False, server_default="false")
 
+    @validates("approval_date")
+    def send_approval_confirmation(self, key, new_approval_date):
+        """Send this user an approval email if their account has just been approved"""
+        if self.approval_date is None and new_approval_date is not None:
```
Comment (scvannost) [inline cidc_api/models/models.py:259]: what should happen if `self.approval_date is None`? especially if `new_approval_date is not None`

**Review by scvannost — APPROVED** [review]

File: .env [hunk 2]
```diff
@@ -9,7 +9,7 @@ AUTH0_CLIENT_ID='Yjlt8LT5vXFJw1Z8m8eaB5aZO26uPyeD'
 PRISM_ENCRYPT_KEY='key'
 
 ENV='dev'
-DEBUG=True
+DEBUG=False
```
Comment (curlup) [inline .env:12]: ? this is dev right?

**Review by curlup — APPROVED** [review]

File: .env (hunk 2 above)
Comment (jacoblurye) [reply .env:12]: Oh, whoops! Left over from local testing - thanks.

File: cidc_api/models/models.py (hunk 1 above)
Comment (jacoblurye) [reply cidc_api/models/models.py:259]: It should send the confirmation email in that case (not sure I understand the question?).

Comment (scvannost) [reply cidc_api/models/models.py:259]: `self._approval_date is not None`, sorry

**Pull request merged**